definition/IRoom.ts

```diff
@@ -38,6 +38,9 @@ export interface IRoom extends IRocketChatRecord {
 			balance: number;
 		}[];
 	};
+
+	isChatOnHold?: boolean;
+	canPlaceChatOnHold?: boolean;
```

Comment:
It looks permission, not a property.

Reply:
Any suggestions for this property name?

it is for controlling whether the `On Hold` button is visible on the visitorInfo panel

Reply:
Actually no, the abandonment setting has nothing to do with this feature, its completely independent. 

![image](https://user-images.githubusercontent.com/34130764/111359899-1cc67180-86b2-11eb-97a1-74a7be1f993e.png)
There is one setting to control which chat is considered as abandoned so that we could perform some action on it(like close it or put it on-hold). :top: This is that setting

The other setting which `canPlaceChatOnHold` is referring to, is to control when an agent is able to see the On-Hold button via which they will be able to manually place the chat on-hold. :arrow_down: This is the setting controlling that behaviour
![image](https://user-images.githubusercontent.com/34130764/111360220-7dee4500-86b2-11eb-8cd1-15392f50e768.png)

Reply:
I wasn't talking about a setting, I was talking about a property, this one:
`omnichannel.predictedVisitorAbandonmentAt`

So, if we set a property to predict abandonment, why don't we do the same with on-hold?
As far as I know, an agent can place a chat on-hold only after abandonment, right?  If so, I noticed that there's a setting(timeout) to define a chat ready to be placed on-hold, which means that rather than just setting a boolean flag on the room, we could add a real information there, and on the client-side we would just check if the timestamp to allow chat on-hold has been reached.

That's my point.

Reply:
Removed this property as its no longer required